src/go/rpk/pkg/cli/cmd/topic/autorestore/start.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+package autorestore
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newStartCommand(fs afero.Fs) *cobra.Command {
+	var wait bool
+	var pollingInterval uint
+
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "Start the autorestore process",
+		Long:  ``,
+		Run: func(cmd *cobra.Command, args []string) {
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			client, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			err = client.StartAutomatedRecovery(cmd.Context())
+			var he *admin.HTTPResponseError
+			if errors.As(err, &he) {
+				if he.Response.StatusCode == 404 {
+					body, bodyErr := he.DecodeGenericErrorBody()
+					if bodyErr == nil {
+						out.Die("Not found: %s", body.Message)
+					}
+				} else if he.Response.StatusCode == 400 {
+					body, bodyErr := he.DecodeGenericErrorBody()
+					if bodyErr == nil {
+						out.Die("Cannot start auto-restore: %s", body.Message)
+					}
+				}
+			}
+
+			out.MaybeDie(err, "error starting auto-restore: %v", err)
+			fmt.Printf("Successfully started auto-restore\n")
+
+			if !wait {
+				return
+			}
+			pollingIntervalSeconds := time.Duration(pollingInterval) * time.Second
+
+			fmt.Printf("Waiting for auto-restore to complete...\n")
```

Comment:
I think fmt.Println is fine here and above

Reply:
Good catch!